.goreleaser.yml

```diff
@@ -16,37 +16,50 @@ builds:
     - arm64
     - ppc64le
     - s390x
+  - id: unpack
```

Comment:
Could we create a follow-up issue to switch from the pod based unpacking method to the direct image registry client approach? Using it would eliminate the need to do all this effort to include this `unpack` binary in the images.

Reply:
I think it would be _ideal_ to use it as part of this PR, but I don't want to make you have to rewrite a bunch of logic since we have a lot of future work held on getting this helm work in.

Reply:
Yep, created the issue here: https://github.com/operator-framework/operator-controller/issues/853